components/elm/src/biogeochem/ComputeSeedMod.F90

```diff
@@ -183,17 +183,17 @@ subroutine LeafProportions(pft_type, ignore_current_state, &
     pstorage = 0._r8
     pxfer    = 0._r8
 
-    if (tot_leaf == 0._r8 .or. ignore_current_state) then
+    !if (tot_leaf == 0._r8 .or. ignore_current_state) then
        if (veg_vp%evergreen(pft_type) == 1._r8) then
           pleaf    = 1._r8
        else
           pstorage = 1._r8
        end if
-    else
-       pleaf    = leaf        /tot_leaf
-       pstorage = leaf_storage/tot_leaf
-       pxfer    = leaf_xfer   /tot_leaf
-    end if
+    !else
+    !   pleaf    = leaf        /tot_leaf
+    !   pstorage = leaf_storage/tot_leaf
+    !   pxfer    = leaf_xfer   /tot_leaf
+    !end if
```

Comment:
let's delete these commented-out lines and reindent the remaining lines.

Reply:
@dmricciuto please see above.

Reply:
@dmricciuto lease take care of above comment.